stmhal/boards/ESPRUINO_PICO/mpconfigboard.h

```diff
@@ -2,6 +2,12 @@
 #define MICROPY_HW_MCU_NAME         "STM32F401CD"
 #define MICROPY_PY_SYS_PLATFORM     "pyboard"
 
+#undef  MICROPY_PY_USOCKET
```

Comment:
Ok, yes, we'd then need #ifndef in mpconfigport.h for those options that are configurable.